doc/cephadm/services/mon.rst

```diff
@@ -170,6 +170,59 @@ network ``10.1.2.0/24``, run the following commands:
 
     ceph orch apply mon --placement="newhost1,newhost2,newhost3" 
 
+
+Setting Crush Locations for Monitors
+------------------------------------
+
+Cephadm supports setting crush locations for mon daemons
+using the mon service spec. The crush locations are set
+by hostname. When cephadm deploys a mon on a host that matches
+a hostname specified in the crush locations, it will add
+--set-crush-location <crush-location> where the crush location
+is the first entry in the list of crush locations for that
+host. If multiple crush locations are set for one host, cephadm
+will attempt to set the additional locations using the
+"ceph mon set_location" command.
+
+.. note::
+
+   Setting the crush location in the spec is the recommended way of
```

Comment:
Maybe make an info box / note that tiebreakers are only for stretch-mode clusters? Aren't they?

Reply:
I think so. I added a note that mentions they're part of stretch mode clusters with a link to the stretch mode docs.